Gemfile

```diff
@@ -18,13 +18,17 @@ group :development do
     gem 'simplecov-console', '~> 0.6'
   end
 
-  if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.5.0')
-    gem 'puppet', '~> 6.0'
-  elsif Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.4.0')
-    gem 'puppet', '~> 5.0' # rubocop:disable Bundler/DuplicatedGem Nope, not a duplicate!
-  else
-    gem 'puppet', '~> 4.0' # rubocop:disable Bundler/DuplicatedGem Nope, not a duplicate!
-  end
+  puppet_version = if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.7.0')
```

Comment:
Thinking more about this: why is `RUBY_VERSION.dup` used? It's not like we modify anything. I opened https://github.com/puppetlabs/puppet-modulebuilder/pull/33.